packages/gatsby/static/configuration/manifest.json

```diff
@@ -288,6 +288,10 @@
           "enum": ["public", "restricted"],
           "examples": ["public"]
         },
+        "type": {
+          "$ref": "#/properties/type",
+          "description": "Same principle as the `publishConfig.type` property; this value will be used instead of the top-level `type` field when generating the workspace tarball."
```

Comment:
That's a reference to itself (self-reference)

Reply:
I think that's a reference to https://github.com/yarnpkg/berry/pull/4204/files#diff-b98641d8f09abf73c08b54295fb7e4fbd50cd4ea2275b602e4659de318865433R32-R37

Reply:
The description says that `publishConfig.type` is the "same principle as `publishConfig.type`" which is a self-reference and doesn't say much. I'm guessing you meant to point to `publishConfig.bin`

Reply:
Uh oh, yes yes I see